src/ESLintLinter.php

```diff
@@ -84,6 +84,10 @@ public function getLinterConfigurationOptions() {
         'type' => 'optional bool',
         'help' => pht('Specify whether eslint should autofix issues. (https://eslint.org/docs/user-guide/command-line-interface#fixing-problems)'),
       ),
+      'eslint.cwd' => array(
```

Comment:
Will this be / should this be a different directory from the [configuration option already provided by `NodeExternalLinter`](https://github.com/pinterest/arcanist-linters/blob/227bfee6483b998eb09fb7ff45eeb7ba22abbbb5/src/NodeExternalLinter.php#L51-L56)?

If so, you should give this a unique name so that it does not conflict (and does not break expectations for what this value should be).

If not, the [configuration value will already be set within `$this->cwd`](https://github.com/pinterest/arcanist-linters/blob/227bfee6483b998eb09fb7ff45eeb7ba22abbbb5/src/NodeExternalLinter.php#L64-L66), so you could leave `setLinterConfigurationValue` as you have it (process the value twice) or perhaps customize `getDefaultFlags` instead.

Reply:
Updated to pull cwd from parent, and moved to mandatory flags because it's not conditional anymore.